build/update-attribution-files/create_pr.sh

```diff
@@ -195,4 +210,21 @@ for FILE in $(find . -type f \( -name Help.mk -o -name Makefile \)); do
     pr::file:add $FILE
 done
 
+# Not handling go.sum/go.mod yet in eks-a
+# stash go.sum files
+#git stash --keep-index
+
 pr::create::help
+
+# git checkout $MAIN_BRANCH
+
+# if [ "$(git stash list)" != "" ]; then
+#     git stash pop
+# fi
+
+# # Add go.mod files
+# for FILE in $(find . -type f \( -name go.sum -o -name go.mod \)); do    
+#     git check-ignore -q $FILE || git add $FILE
+# done
+
+# pr::create::go-mod
```

Comment:
Not sure what all this is for

Reply:
on eks-d, we c heck in the go.sum/go.mod files so we can get denpendabot alerts for go deps that have security updates. id like to add that to this repo, but have been heistant just because of the wide scope due to the number of projects. I do want to plan for it and do it at some point tho.

When i sync the code from eks-d and here i tend to try and leave as much code in as possible even if the repo doesnt use it directly, makes syncing the next time a bit easier